nexus-repository-composer-it/src/test/it-resources/packages.json

```diff
@@ -0,0 +1 @@
+{"packages":[],"notify":"https://packagist.org/downloads/%package%","notify-batch":"https://packagist.org/downloads/","providers-url":"/p/%package%$%hash%.json","metadata-url":"/p2/%package%.json","search":"https://packagist.org/search.json?q=%query%&type=%type%","provider-includes":{"p/provider-2013$%hash%.json":{"sha256":"cab932951e91e2b9ec613ef883adc38ed52a48f9e23e8a2baa02ec49b650bbb3"},"p/provider-2014$%hash%.json":{"sha256":"d0401e6a04adab24c42c34d83e2009b2cf693265727b596634abf86edb72261f"},"p/provider-2015$%hash%.json":{"sha256":"3c56c6be8b4e8d9afbedd4da4fde7cbea7461961efe4de56897eb82d2811042f"},"p/provider-2016$%hash%.json":{"sha256":"484e71121232e53874430a6c53394218da3e552cda4d8bcca0a61d613efd27fb"},"p/provider-2017$%hash%.json":{"sha256":"55b9009c4c8190343d3c9578893d221979d01b1932275bf28b308309c32fa3ca"},"p/provider-2018$%hash%.json":{"sha256":"93f158c2b271d16bff7f9c2eec7c10820cedba5cf5db4f8eac9eb66063fd7df2"},"p/provider-2019-01$%hash%.json":{"sha256":"349cd385968423a155088c8c2935004274715aee5de6280c6fa12c18a4ea7c15"},"p/provider-2019-04$%hash%.json":{"sha256":"c64ecd2711550f1aff08be79f8a1429f9081dc7ec3dbe242d531bb664026c844"},"p/provider-2019-07$%hash%.json":{"sha256":"84d28cbd7676aaa7151a835abb5cc4c887c80de984be70f1b09d3f5823477bea"},"p/provider-archived$%hash%.json":{"sha256":"732a818b5d8ef852b04d73ac2a5685ad369f4debfe1a0d36f237e62f39b688de"},"p/provider-latest$%hash%.json":{"sha256":"a22f4df6424192a11a32d5a9bcd475f14c4a83595746f57a391f208b6144a41f"}}}
```

Comment:
Trivial: newline

Reply:
in: https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/48/commits/19d186fef09cda04d4d700e0caddb4e1d2d5092c

dang, didn't know intellij's "add new line on save" doesn't work for json -- learn something new everyday!